an/v1/AN_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,2464 @@
+              properties:
+                UNLocationCode:
+                  type: string
+                  pattern: '^[A-Z]{2}[A-Z2-9]{3}$'
+                  minLength: 5
+                  maxLength: 5
+                  description: |
+                    The UN Location code specifying where the place is located. The pattern used must be
+
+                    - 2 characters for the country code using [ISO 3166-1 alpha-2](https://www.iso.org/obp/ui/#iso:pub:PUB500001:en)
+                    - 3 characters to code a location within that country. Letters A-Z and numbers from 2-9 can be used
+
+                    More info can be found here: [UN/LOCODE](https://unece.org/trade/cefact/UNLOCODE-Download)
+                  example: NLAMS
+              required:
+                - UNLocationCode
+            - type: object
+              title: Free text
+              properties:
+                freeText:
+                  type: string
+                  maxLength: 35
+                  description: |
+                    The name of the location where payment will be rendered by the customer.
+                  example: DCSA Headquarters
+              required:
+                - freeText
+        utilizedTransportEquipments:
+          description: |
+            List of `Utilized Transport Equipment`.
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/UtilizedTransportEquipment'
+        consignmentItems:
+          description: |
+            List of `Consignment`.
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/ConsignmentItem'
+      required:
+        - issueDate
+        - carrierCode
+        - carrierCodeListProvider
+        - carrierContactInformation
+        - transportDocumentReference
+        - transportDocumentTypeCode
+        - isElectronic
+        - deliveryTypeAtDestination
+        - cargoMovmentTypeAtDesination
+        - termsAndConditions
+        - documentParties
+        - transports
+        - utilizedTransportEquipments
+        - consignmentItems
+
+    ######################
+    # Party Contact Detail
+    ######################
+    PartyContactDetail:
+      title: Party Contact Detail
+      description: |
+        The contact details of the person to contact. It is mandatory to provide either `phone` and/or `email` along with the `name`.
+      example:
+        name: Henrik
+        phone: +45 51801234
+      properties:
+        name:
+          type: string
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 100
+          description: |
+            Name of the contact
+          example: Henrik
+      anyOf:
+        - title: Phone required
+          type: object
+          properties:
+            phone:
+              type: string
+              pattern: ^\S+(\s+\S+)*$
+              maxLength: 30
+              description: |
+                bla bla bla
```

Comment:
The phone description is - 'Phone number of the party' in the IFS

Reply:
Fixed + also fixed missing email description